.gitignore

```diff
@@ -7,6 +7,7 @@ package-lock.json
 yarn.lock
 /es
 /lib
+/.idea
```

Comment:
This doesn't fall in the scope of this PR, can you remove it?

Reply:
@WikiRik - yes i can remove as this came in because of Intellij IDE

Reply:
Done